eks/mng/config-update/config-update.go

```diff
@@ -0,0 +1,243 @@
+// Package configupdate implements EKS cluster config update tester.
+package configupdate
```

Comment:
For now, let's not worry about other config update. And put these code under `eks/mng/scale`?